introduction/site_survey.md

```diff
@@ -0,0 +1,353 @@
+
+<a name="default_internal_network_ranges"></a>
+## 5. Default Internal Network Ranges 
+
+The initial installation of the system creates default networks with default settings and with no external exposure.
+These IP address default ranges ensure that no nodes in the system attempt to use the same IP address as a
+Kubernetes service or pod, which would result in undefined behavior that is extremely difficult to reproduce or
+debug.
+
+The following table shows the default IP address ranges
+
+| Network | Default IP Address Range | Site Value (if not default) |
+| --- | --- | --- | 
+| Kubernetes service network | 10.16.0.0/12 | |
+| Kubernetes pod network | 10.32.0.0/12 | |
+| Install Network (MTL) | 10.1.0.0/16 | |
+| Node Management Network (NMN) | 10.252.0.0/17 | |
+| High Speed Network (HSN) | 10.253.0.0/16 | |
+| Hardware Management Network (HMN) | 10.254.0.0/17 | |
+| Mountain NMN Allocate a /22 from this range per liquid cooled cabinet: * cabinet 1 * cabinet 2 * cabinet 3 * ...  | 10.100.0.0/17 Example IP address in the allocated ranges: * 10.100.0.0/22 * 10.100.4.0/22 * 10.100.8.0/22 * ... | |
+| Mountain HMN Allocate a /22 from this range per liquid cooled cabinet: * cabinet 1 * cabinet 2 * cabinet 3 * ... | 10.104.0.0/17 Example IP address in the allocated ranges: * 10.104.0.0/22 * 10.104.4.0/22 * 10.104.8.0/22 * ... | |
+| River NMN | 10.106.0.0/17 | |
+| River HMN | 10.107.0.0/17 | |
+
+
+<a name="SHCD"></a>
+## 6. SHCD
+
+The Shasta Cabling Diagram (SHCD) is a multiple tab spreadsheet prepared by HPE Cray Manufacturing with information about the components
+in an HPE Cray EX system. This document has much information about the system. Included in the SHCD are a configuration summary with
+revision history, floor layout plan with cabinet ID numbers, type and location of components in the air-cooled cabinets, type and location of components in the
+liquid-cooled cabinets, device diagrams for switches and nodes in the cabinets, list of source and destination of every HSN cable,
+list of source and destination of every cable connected to the spine switches, list of source and destination of every cable connected
+to the NMN, list of source and destination of every cable connected to the HMN. list of cabling for the KVM, and routing of power to the PDUs.
+
+The installation of CSM software requires that the SHCD be available.  Some information will be manually collected from the SHCD, but some of the tabs can be extracted into CSV formatted files to be fed to automatic configuration tools.
+
+<a name="csi_command_line_configuration_payload"></a>
+## 7. `csi` Command Line Configuration Payload
+
+This information from a site survey can be given to the `csi` command as command line arguments.
+The information is shown here to explain what data is needed.
+
+The air-cooled cabinet is known to `csi` as a `river` cabinet. The liquid-cooled cabinets are either
+`mountain` or `hill` (if a TDS system).
```

Comment:
Should define (or link to definition of) TDS system here. I've had folks ask me what that meant when seeing it in our docs.

Reply:
There is a glossary for all of docs-csm.  I'm not going to link "TDS" here to the glossary.  So many terms in this site survey are not explained upon first usage.